.github/updatecli/updatecli.d/wiremock.yaml

```
@@ -0,0 +1,66 @@
+name: 'docs: bump wiremock version used in the documentation'
```

I would also update Dockerfiles, etc.

```suggestion
name: 'Bump WireMock version'
```